src/locale/en.ts

```diff
@@ -9,6 +9,7 @@ export const en = {
     multiline: 'Multiline',
     publicKeyQrCode: 'Public key QR code',
     privateKey: 'Private key',
+    privateKeyWIF: 'Private key in WIF',
```

Comment:
Before merge, please add translations or at least placeholders. We don't want the users to see typical for JS "undefined"